src/config/graphical-session/session-management.md

```diff
@@ -0,0 +1,18 @@
+# Session and Seat Management
+
+Session and seat management is not necessary for every setup, but it can be used
+to safely create temporary runtime directories, provide access to hardware
+devices, and control system shutdown.
+
+## elogind
+
+[elogind(8)](https://man.voidlinux.org/elogind.8) is a standalone version of
+`systemd-logind`, a service to manage user logins. This service provides
+necessary features for most desktop environments and Wayland compositors. To
+make use of its features, install the `elogind` package:
+
+```
+# xbps-install -S elogind
```

Comment:
As mentioned in #190 we are, in fact, avoiding sharing this command every time. :)

Reply:
In my defense, it's legacy documentation :p

Reply:
I removed the install command, but didn't like the package installation page. Do you think that's necessary?